docs/data/material/components/dividers/DividerMaterialYouPlayground.js

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import Divider from '@mui/material-next/Divider';
+import MaterialYouUsageDemo from 'docs/src/modules/components/MaterialYouUsageDemo';
+import {
+  Avatar,
+  List,
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  Grid,
+} from '@mui/material';
+import ImageIcon from '@mui/icons-material/Image';
+import WorkIcon from '@mui/icons-material/Work';
+
+export default function DividerMaterialYouPlayground() {
+  const [showLabel, setShowLabel] = React.useState(false);
+  const [orientation, setOrientation] = React.useState('horizontal');
+
+  return (
+    <MaterialYouUsageDemo
+      componentName="Divider"
+      data={[
+        {
+          propName: 'variant',
+          knob: 'select',
+          options: ['fullWidth', 'inset', 'middle'],
+          defaultValue: 'fullWidth',
+        },
+        {
+          propName: 'orientation',
+          knob: 'select',
+          options: ['horizontal', 'vertical'],
+          defaultValue: 'horizontal',
+          onChange: (e) => setOrientation(e.target.value),
+        },
+        {
+          propName: 'showLabel',
+          knob: 'switch',
+          defaultValue: false,
+          onChange: () => {
+            setShowLabel((prev) => !prev);
+          },
+        },
```

Comment:
I think we should remove this one because it might be confusing, as `showLabel` is not a Divider prop. We don't want to mislead people.